README.md

```diff
@@ -75,6 +87,8 @@ In order to run all example in this repository you need to have
   - dictionary
```

Comment:
Ow, actually its my fault not catch this before. But can you make its first word uppercase? Would be appreciated.